### DISCUSSION TITLE
Is Copilot currently bugged?

### DISCUSSION THREAD

**Discussion by igniscyan** [community/community#51213] [Copilot Conversations]
### Select Topic Area

Question

### Body

I feel like in recent months, Copilot has degraded in terms of suggestion frequency and quality. In VSCode and nvim, Copilot seems to only suggest one line at a time and rarely generates code based on comments like it used to. I've seen threads about this, but no definitive answers.

Is something up with Copilot? Currently, as a paid service, it went from feeling like an insane value to no longer worth it.

**Comment by halcwb** [comment]
I just tried it for F# and it makes only nonsensical suggestions. I am going to quickly cancel my free trial if this doesn't imporve.

**Comment by iamcjwilliams** [comment]
@igniscyan Can you provide a screenshot of your issue?

**Comment by brucelightyear** [comment]
Same thing for me, the quality of the recommendations is deteriorating. Even for several days, I only have one or two lines in recommendations and absolutely none in the side panel.

**Reply by brucelightyear** [reply]
I forgot, it recommends me my own code written above 🤣

**Reply by st4s1k** [reply]
same for me
